deploy/gk-deploy

```diff
@@ -129,7 +129,7 @@ eval_output() {
   )
 }
 abort() {
-  ${CLI} delete all,service,deployment,secret --selector="heketi"
+  ${CLI} delete service,deployment,pod --selector="heketi"
```

Comment:
Yes, because I'm relabeling the pod with the "heketi" label.

"all" is removed because we don't have any resources labeled "heketi" to delete that aren't service, deployments, or pods.

But I'm seeing some other changes that can be made, so I'll resubmit.